groups/apps.py

```diff
@@ -0,0 +1,16 @@
+from django.apps import AppConfig
+
+
+class GroupsConfig(AppConfig):
+    """
+    Configuration for `groups`.
+
+    Provides:
+    * default_active_threshold_days - a user is considered not to be active if they
+      haven't made a comment within the past [this value] days, and vice versa.  A
```

Comment:
Rather than "not active when haven't commented" this could be clearer as:

> ...a user is considered to be active if they have made a comment within the past...